bon-macros/src/builder/builder_gen/top_level_config/mod.rs

```diff
@@ -75,6 +75,12 @@ pub(crate) struct TopLevelConfig {
     /// Specifies the derives to apply to the builder.
     #[darling(default, with = crate::parsing::parse_non_empty_paren_meta_list)]
     pub(crate) derive: DerivesConfig,
+
+    #[darling(default)]
+    pub(crate) build_from: bool,
```

Comment:
These should use `Option<Option<ItemSigConfig>>` to make their names, visibility and docs configurable, plus inherit the visibility of the builder struct similar to how the regular `finish_fn` does that, otherwise the docs would be incorrectly claiming that these attributes support `vis`, `name`, `doc` attributes.

I think this may require writing a custom `with` parser function that accepts a `syn::Meta::Path` (in which case `None` (inner None), and otherwise delegates to `ItemSigConfig`.